### PR TITLE
AK-24311 Update available size for various connectors and service

### PR DESCRIPTION
**Pull request by spikewang**
Extend available size options for various connector and service.

### CODE REVIEW

File: alkira/resource_alkira_connector_ipsec.go [hunk 1]
```diff
@@ -58,9 +58,10 @@ func resourceAlkiraConnectorIPSec() *schema.Resource {
 							Optional: true,
 						},
 						"enable_tunnel_redundancy": {
-							Description: "indicates to Alkira that health for the site should be reported as good as long as any one of the tunnels is up.",
-							Type:        schema.TypeBool,
-							Optional:    true,
+							Description: "indicates to Alkira that health for the site should " +
```
Comment (mac-alkira) [inline alkira/resource_alkira_connector_ipsec.go:61]: indicates should be capitalized

File: alkira/resource_alkira_connector_ipsec.go [hunk 2]
```diff
@@ -281,10 +282,10 @@ func resourceAlkiraConnectorIPSec() *schema.Resource {
 				Required:    true,
 			},
 			"size": &schema.Schema{
-				Description:  "The size of the connector. one of `SMALL`, `MEDIUM` and `LARGE`.",
+				Description:  "The size of the connector. one of `SMALL`, `MEDIUM`, `LARGE`, `2LARGE`, `4LARGE`, `5LARGE`, `10LARGE` and `20LARGE`.",
```
Comment (mac-alkira) [inline alkira/resource_alkira_connector_ipsec.go:285]: Based on previous comments the first period in this description should be a comma.

**Review by mac-alkira — APPROVED** [review]
Minor doc changes. Otherwise approved.

File: alkira/resource_alkira_connector_ipsec.go (hunk 1 above)
Comment (spikewang) [reply alkira/resource_alkira_connector_ipsec.go:61]: Yeah, good point indeed, let me find if any better wording for this...

File: alkira/resource_alkira_connector_ipsec.go (hunk 2 above)
Comment (spikewang) [reply alkira/resource_alkira_connector_ipsec.go:285]: Good catch 👍

**Review by mac-alkira — APPROVED** [review]

**Pull request merged**